pato-base.obo

```diff
@@ -11929,35 +11933,43 @@ relationship: is_opposite_of PATO:0001742 ! increased radioactivity
 
 [Term]
 id: PATO:0001744
-name: radiation exposure
-def: "A radiation quality inhering in a bearer by virtue of what the bearer receives as a result of being exposed to ionizing radiation." [hps:hps]
+name: irradiated
+def: "A quality of a substance inhering in a bearer by virtue of it being currently or previously exposed to electromagnetic radiation," [https://orcid.org/0000-0002-5111-7263, ISSN:9781496335418]
+comment: This may refer to incident external irradiation or internal radiation from an ingested or otherwise internal source.
 subset: attribute_slim
 subset: scalar_slim
-is_a: PATO:0001739 ! radiation quality
+synonym: "radiation exposed" EXACT []
+is_a: PATO:0002198 ! quality of a substance
 
 [Term]
 id: PATO:0001745
-name: radiation absorbed dose
-def: "A radiation exposure quality inhering in a substance by virtue of the radiation energy \"deposited\" in a kilogram of a substance." [arpansa:arpansa]
+name: obsolete radiation absorbed dose
+def: "OBSOLETE A radiation exposure quality inhering in a substance by virtue of the radiation energy \"deposited\" in a kilogram of a substance." [arpansa:arpansa]
+comment: see: https://github.com/pato-ontology/pato/issues/485
```

Comment:
dont know who did that but please just use term tracker item, not this weird construct, in the future

Reply:
whoops that's me, sorry about that >.< will fix it for next release